cloudfoundry-client-lib/src/main/java/org/cloudfoundry/client/lib/rest/CloudControllerRestClientImpl.java

```diff
@@ -252,6 +253,7 @@
     private static final String DEFAULT_HOST_DOMAIN_SEPARATOR = "\\.";
     private static final String DEFAULT_PATH_SEPARATOR = "/";
     private static final long JOB_POLLING_PERIOD = TimeUnit.SECONDS.toMillis(5);
+    private static final int BATCH_SIZE_FOR_FILTER_IN_METADATA_REQUESTS = 100;
```

Comment:
Is there a character limit for application/service instance names? Is it possible that 100 names could exceed the 8000 character URL limit of CF?

Reply:
The PR also introduces a change in the requests -> now - for applications, we use the UUID instead of the application name, as the name does not seem to have a limit (I failed to find one or reproduce an error with long app names?), and we know the UUIDs max length of 32. For service instances we use the name (limit of 50 characters). For both scenarios the batch size of 100 seems to be sufficient in not exceeding the limit of 8190 which seems to be the CF default (same as apache documented default at http://httpd.apache.org/docs/2.2/mod/core.html#limitrequestline)

Reply:
Cool.